var/spack/repos/builtin/packages/kahip/package.py

```diff
@@ -46,17 +46,25 @@ class Kahip(Package):
     url       = 'http://algo2.iti.kit.edu/schulz/software_releases/KaHIP_2.00.tar.gz'
 
     version('develop', git='https://github.com/schulzchristian/KaHIP.git')
-    version('2.00', '9daeda32f43c90570ed436d5d93c8a872b1a14d8')
+    version('2.00', '0a66b0a604ad72cfb7e3dce00e2c9fdfac82b855')
```

Comment:
Confirmed with @schulzchristian that this checksum change is due to a patch without a version number increment.